### PR TITLE
Add support for dotnet core > 3.0, dotnet 46, 47, 48

### LINKED ISSUE #153: Plugin not Working with wpf (.Net Core 3.0)
## Expected Behavior

Selecting File without exception

## Actual Behavior

Exception when calling 
var filePicker = await CrossFilePicker.Current.PickFile();

Ex: 
System.NotImplementedException
  HResult=0x80004001
  Message=This functionality is not implemented in the portable version of this assembly. You should reference the NuGet package from your main application project in order to reference the platform-specific implementation.
  Source=Plugin.FilePicker
  StackTrace:
   at Plugin.FilePicker.CrossFilePicker.get_Current()
   at OnObject.ImportPage.<SelectUploadButton_Clicked>d__10.MoveNext() 

## Steps to Reproduce the Problem

  1. Add Wpf Core 3.0 Project to your xamarin app
  1. Open Filepicker
  

## Specifications

  - NuGet package version: 2.1.34
  - Platform(s): Windows 10

### LINKED ISSUE #186: Not working on Windows 10 IoT Core
## Expected Behavior
Open a dialog to explore file.

## Actual Behavior
await CrossFilePicker.Current.PickFile() instantly return null value (it's working on uwp runned on regoular windows)

## Steps to Reproduce the Problem

  1. just run "await CrossFilePicker.Current.PickFile()" from a windows 10 iot device

## Specifications

  - NuGet package version: 2.1.34
  - Platform(s): Windows 10 IoT Core

### PR DESCRIPTION
**Pull request by de-wolff**
### Description of Change ###

Create support for dotnetcore

Add dotnetcore target frameworks, and link them to dotnet 45 sources

### Issues Resolved ### 

- fixes #186 Not working on Windows 10 IoT Core
- fixes #153 Plugin not Working with wpf (.Net Core 3.0)

### Platforms Affected ### 

- dotnetcore 3.0
- dotnetcore 3.1
- dotnet 4.6
- dotnet 4.7
- dotnet 4.8

### CODE REVIEW

**Comment by de-wolff** [comment]
I added samples for all WPF framework versions, and they all did run here.

I did a test  with one winform on the .dotnet core framework, but I am not really familiar with the winform environment, so it wil take same time to have good samples available

File: src/Plugin.FilePicker/Plugin.FilePicker.csproj [hunk 1]
```diff
@@ -79,6 +83,10 @@
     <Compile Include="**\*.netstandard.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp3')) ">
+    <Compile Include="**\*.net45.cs" />
```
Comment (VladislavAntonyuk) [inline src/Plugin.FilePicker/Plugin.FilePicker.csproj:87]: is it correct?

**Comment by jfversluis** [comment]
This is great @de-wolff thank you so much!

Could you have a look at the comment by @VladislavAntonyuk ? It seems a bit odd indeed. Thanks :)

File: src/Plugin.FilePicker/Plugin.FilePicker.csproj (hunk 1 above)
Comment (de-wolff) [reply src/Plugin.FilePicker/Plugin.FilePicker.csproj:87]: Yes,
I know it looks a bit odd, and possible a refactoring should be done somewhere in the future, but there is a parallel development for dot net core and dot net standard, and dot net framework.

.net core 3.0 most important change compared to .net core 2.x is that it added support for the WPF framework and for forms. As the .net framework version was 4.5 at the moment of releasing .net core 3.0, the WPF and Forms implementation of .net core 3.0 is almost the same as the WPF and Forms implementation of .net framework 4.5.
As a result of this , when a WPF or Form implementation is available, most of the time the .net core implementation can be done, just by using the .net framework 4.5 implementation.

I hope this explains this odd lines in the project file

Jaap

Comment (jfversluis) [reply src/Plugin.FilePicker/Plugin.FilePicker.csproj:87]: Awesome. That totally makes sense. I'll merge it and make a beta package out of it to get some feedback on this. A BIG thank you for this wonderful contribution

**Review by jfversluis — APPROVED** [review]

**Pull request merged**